docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
@defektive jeah :) maybe also add a `msfvenom.sh` script to the docker folder with this command in it so it's more user friendly